keymanager-api-common/src/api.rs

```diff
@@ -98,25 +99,34 @@ pub struct ReplicateResponse {
 
 /// Request runtime/key pair id tuple.
 #[derive(Clone, Default, cbor::Encode, cbor::Decode)]
-pub struct RequestIds {
+pub struct RuntimeKeyRequest {
```

Comment:
Maybe this should be something like `LongTermKeyRequest` to contrast with ephemeral ones? As all keys generated by the key manager are "runtime keys".